src/main/resources/assets/js/application.js

```diff
@@ -0,0 +1,25 @@
+function getHostName(url) {
+  var match = url.match(/:\/\/(www[0-9]?\.)?(.[^/:]+)/i);
```

Comment:
> document.body.className = url.(s.indexOf("gov.uk") == -1) ? "open-registers" : "gov-uk";

The `url.` seems like a typo but otherwise I agree that we don't need to make it work with any possible domain, and that a quick ternary like this is sufficient.